### PR TITLE
Add HASS configurator to core add-ons

### PR DESCRIPTION
**Pull request by danielperna84**
Requesting to add the HASS configurator add-on to the core repository.

Documentation: https://github.com/home-assistant/home-assistant.github.io/pull/3441

### CODE REVIEW

File: hass-configurator/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM %%BASE_IMAGE%%
```
Comment (pvizeli) [inline hass-configurator/Dockerfile]:
```
ARG BUILD_FROM
FROM $BUILD_FROM
```
The anounce for new build system go out in next 24 hours :)

File: hass-configurator/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+# Add env
+ENV LANG C.UTF-8
+
+# Setup base
+RUN apk add --no-cache python3 tzdata git \
```
Comment (pvizeli) [inline hass-configurator/Dockerfile]: `tzdata` is not need anymore (include in base image)

File: hass-configurator/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# HASS Configurator (Hass.IO add-on)
```
Comment (pvizeli) [inline hass-configurator/README.md]: Please remove readme, that going into http://hass.io/addons/...

File: hass-configurator/config.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    "startup": "application",
+    "webui": "http://[HOST]:[PORT:3218]",
+    "boot": "auto",
+    "ports": {
+      "3218/tcp": 3218
+    },
+    "map": ["config:rw", "ssl"],
+    "options": {
+      "BASEPATH": "/config",
+      "HASS_API": "http://172.17.0.1:8123/api/",
```
Comment (pvizeli) [inline hass-configurator/config.json]: hassio have a own network so you can use names:
`http://homeassistant:8123/api/`

File: hass-configurator/run.sh [hunk 5]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+CONFIG_PATH=/data/options.json
+
+python3 /configurator.py ${CONFIG_PATH}
```
Comment (pvizeli) [inline hass-configurator/run.sh]: You need not have a `run.sh` or modify the `configurator.py` to +a. Add this into Dockerfile and remove this `run.sh` file:
```
CMD ["python3", "/configurator.py", "/data/options.json"]
```

**Comment by danielperna84** [comment]
Looking good. I'm going to update the documentation (lower case settings) this evening.

**Pull request merged**